src/transaction.js

```diff
@@ -304,4 +350,50 @@ export class Transaction {
       .toXDR()
       .toString('base64');
   }
+
+  /**
+   * Builds a FeeBumpTransaction
+   * @param {StrKey} feeSource - The account paying for the transaction.
+   * @param {string} baseFee - The max fee willing to pay per operation in inner transaction (**in stroops**). Required.
+   * @param {TransactionV1Envelope} innerTxEnvelope - The TransactionV1Envelope to be bumped by this transaction.
+   * @param {string} networkPassphrase - networkPassphrase of the target stellar network (e.g. "Public Global Stellar Network ; September 2015").
+   * @returns {Transaction}
+   * @ignore tell jsdoc to not show this method for now
+   */
+  static buildFeeBumpTransaction(
```

Comment:
what's the rationale for including this function in the `Transaction` class instead of `TransactionBuilder`? also would it make sense for `buildFeeBumpTransaction()` to take an `innerTx` argument of type `Transaction` instead of `innerTxEnvelope` which has type `TransactionV1Envelope`?

Reply:
It makes sense to have this on the builder. While testing it felt to me like this responsibility should be here mostly since I was testing  how `Transaction` behaves when you pass a fee bump envelope. However, I think we can move this back to the build and split the tests.